### PR TITLE
M sc1 57 18

### PR DESCRIPTION
**Pull request by MorozSergei**
Could yu pleas view my home task

Thank you

### CODE REVIEW

File: bash.exe.stackdump [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Stack trace:
```
Comment (pluhin) [inline bash.exe.stackdump]: could you please remove this file from PR

File: sh.exe.stackdump [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Stack trace:
```
Comment (pluhin) [inline sh.exe.stackdump]: could you please remove this file from PR

File: smoroz/01_HTML/Home task add.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+		<p><code>Hello, World!</code></p>
+		<p>Однако в браузере у нас будет отображаться результат выполнения этой программы:</p>
+		<p><b><font size="5">Hello, World!</font></b></p>
+		<hr>
+		<h2>Комментарии и литералы(мнемоники) в HTML</h2>
+		<p>&lt;!‐‐ это комментарий ‐‐&gt;
+		<br>
+		&lt;!‐‐ и это также комментарий,
+		<br>
+		занимающий более одной строки ‐‐&gt;</align></p>
```
Comment (pluhin) [inline smoroz/01_HTML/Home task add.html]: align - what is it here?

File: smoroz/01_HTML/Home task add.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+		<br>
+		обед с 13<u><sup>00</sup></u> до 14<u><sup>00</sup></u>
+		<br>
+		выходной: <u>суббота</u>, <u>воскресенье</u></p>
+		<hr>
+		<h2>Вот такой текст</h2>
+		<h2><sup>Э</sup><sub>т</sub><sup>о</sup> <sup>п</sup><sub>р</sub><sup>о</sup><sub>с</sub><sup>т</sup><sub>о</sub> <sup>в</sup><sub>е</sub><sup>с</sup><sub>л</sub><sup>ы</sup><sub>й</sub> <sup>т</sup><sub>е</sub><sup>к</sup><sub>с</sub><sup>т</sup></h2>
+		<hr>
+		<h2>Акроним и курсив</h2>
+		<p>ТФ: Y<i>(x)</i>=e<i><sup>iπx</sup></i>+log<i><sub>x</i></sub>4
```
Comment (pluhin) [inline smoroz/01_HTML/Home task add.html]: for pi sign use mnemonic

File: smoroz/01_HTML/python_2.html [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+         Вэтом уроке мы рассмотрим ещё больше типов переменных и напишем ещё больше
+         программ.
+  		<h2>Списки</h2>
+   		<p>Давайте рассмотрим другой тип переменных, который называется <cite>списки</cite>. В других
+		языках программирования они называются <cite>массивами</cite>. Если мы обратимся к аналогии с
+		коробками для обуви, то массив (список) представляет собой несколько склеенных друг с
+		другом коробок с одинаковыми вещами. Например, в одной коробке лежат вилки, в
+		другой — ножи, в третьей — ложки. Посмотрим на простой список— список месяцев. Мы
+		запрограммируем его так:</p>
+		<p><code>months =</code>
```
Comment (pluhin) [inline smoroz/01_HTML/python_2.html]: Do not use tag p for non-human text

File: smoroz/01_HTML/python_2.html [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+		<br>
+		<code>['Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec']</code></p>
+		<p>Чтобы создать список, мы заключили все значения в квадратные скобки <strong>( '[' и ']' )</strong>. Мы
+		назвали наш список <cite>'months'</cite>. Чтобы использовать список, мы дадим команду print
+		<strong>months[0]</strong> или <strong>months[1]</strong> (которые выведут на экран <cite>'Jan'</cite> или <cite>'Feb'</cite>). Помните: счет начинается с нуля. Чтобы найти длину списка, можно использовать такую функцию:</p>
+		<p><code>print len(months)</code></p>
+		<p>которая вернёт 12.</p>
+		<p>Ещё один пример списков — категории в поваренной книге. Например...</p>
+		<p><code>categories = ['Main dish','Meat','Fish','Soup','Cookies']</code></p>
```
Comment (pluhin) [inline smoroz/01_HTML/python_2.html]: Do not use tag p for non-human text

File: smoroz/01_HTML/python_2.html [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+		другом коробок с одинаковыми вещами. Например, в одной коробке лежат вилки, в
+		другой — ножи, в третьей — ложки. Посмотрим на простой список— список месяцев. Мы
+		запрограммируем его так:</p>
+		<p><code>months =</code>
+		<br>
+		<code>['Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec']</code></p>
+		<p>Чтобы создать список, мы заключили все значения в квадратные скобки <strong>( '[' и ']' )</strong>. Мы
+		назвали наш список <cite>'months'</cite>. Чтобы использовать список, мы дадим команду print
+		<strong>months[0]</strong> или <strong>months[1]</strong> (которые выведут на экран <cite>'Jan'</cite> или <cite>'Feb'</cite>). Помните: счет начинается с нуля. Чтобы найти длину списка, можно использовать такую функцию:</p>
+		<p><code>print len(months)</code></p>
```
Comment (pluhin) [inline smoroz/01_HTML/python_2.html]: Do not use tag p for non-human text

File: smoroz/01_HTML/python_2.html [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+		<hr>
+
+		<h2>Комментарии</h2>
+		<p>Далее, мы должны обсудить комментарии. Комментарии важны по нескольким причинам.
+		Они не только говорят вам или другим людям, что вы пытаетесь делать, но и напомнят
+		вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ,
+		комментарии станут важными. Также при их помощи вы заставите Python игнорировать
+		какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак <cite>'#'</cite>.
+		Например:</p>
+		<p><code>#</code></p>
```
Comment (pluhin) [inline smoroz/01_HTML/python_2.html]: Do not use tag p for non-human text

File: smoroz/01_HTML/python_3.html [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+		<ul>
+			<li>Программа на Python — часть 1</li>
+			<li>Программа на Python — часть 2</li>
+			<li>Программа на Python — часть 3</li>
+		</ul>
+		<h2>Содержание</h2>
+		<h3>HOW-TO: Программа на Python, Часть 3</h3>
+		<ul>
+			<li><ins>Модули</ins></li>
+			<li><ins>Функции</ins></li>
```
Comment (pluhin) [inline smoroz/01_HTML/python_3.html]: You did not close tag ul here

File: smoroz/01_HTML/task_1.html [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+		<META charset="utf-8">
+		<TITLE>Пример страницы</TITLE>
+	</HEAD> 
+	<!--начало документа -->
+	<BODY> 
+		<H1>Это заголовок</H1>  
+		<P><DEL>Добрый день, <EM>Уважаемые слушатели</EM></DEL>!Сегодня мы с вами будем 
+		рассматривать структуру стандартного <STRONG>HTML</STRONG> файла.</P> 
+		<P>Далее мы будем использовать произвольный текст для наполнения контентом. Не стоит вдумываться в его содержание.</P>
+		<P>«Си́мпсоны» (англ. The Simpsons) — самый длинный мультсериал в истории американского телевидения, длящийся 25 сезонов. Первая мини-серия «Good night» была показана в «Шоу Трейси Ульман» 19 апреля 1987 года. <!--Демонстрация полноценных серий мультсериала началась 17 декабря 1989 года на канале FOX. Выпускается «Gracie Films» для кинокомпании «20th Century Fox». Мультсериал в настоящее время известен телезрителям более чем 100 стран.</P>
```
Comment (pluhin) [inline smoroz/01_HTML/task_1.html]: need to close this paragraph

File: smoroz/02_HTML/Untitled-1.html [hunk 11]
```diff
@@ -0,0 +1,19 @@
+<! DOCTYPE html>
```
Comment (pluhin) [inline smoroz/02_HTML/Untitled-1.html]: This file is totally wrong to many spaces

**Comment by MorozSergei** [comment]
pleas view my work

**Pull request closed without merging**